test/specs/tmpl.js

```diff
@@ -100,8 +100,8 @@ describe('Tmpl', function() {
     expect(render('{ nonExistingVar ? "yes" : "no" }')).to.equal('no')
     expect(render('{ !nonExistingVar ? "yes" : "no" }')).to.equal('yes')
 
-    window.globalVar = 5
-    expect(render('{ globalVar }')).to.equal(window.globalVar)
+    globalVar = 5
```

Comment:
that `globalVar` will be definied on `window` in case of browsers, and on `global` in case of `node/io.js`